src/enterprise/update/isHealthyServer.ts

```diff
@@ -1,11 +1,12 @@
-import createClient, { healthy } from "./client";
+import { URL } from "url";
+import { getHttpStatusCode } from "../../utils/download.utils";
 import serverUrl, { validateUrl } from "./serverUrl";
 
 export async function isHealthyServer(): Promise<boolean> {
   const url = serverUrl();
   if (!validateUrl(url)) {
     return false;
   }
-  const client = await createClient(url as string);
-  return healthy(client);
+
+  return (await getHttpStatusCode(new URL("/health", url).toString())) === 200;
```

Comment:
extract to `ishealthy`

Reply:
why? this is the name of the function (and no one else is using it)

Reply:
bc `ishealthy ` is readable ` (await getHttpStatusCode(new URL("/health", url).toString())) === 200` dose not

Reply:
but the name of this function is `isHealthy`..

Reply:
oh, you meant the `isHealthyServer`, cool